projects/valtimo/dossier/src/lib/components/dossier-process-start-modal/dossier-process-start-modal.component.ts

```diff
@@ -75,6 +81,9 @@ export class DossierProcessStartModalComponent implements OnInit {
     this.processLinkId = null;
     this.formDefinition = null;
     this.formFlowInstanceId = null;
+    this.processService.getProcessDefinitionXml(this.processDefinitionId).subscribe((result) => {
```

Comment:
I'd like to see this call only executed if the feature flag is enable, now it always happens.

Reply:
Added if statement to check for the feature toggle